mne/preprocessing/eyetracking/tests/test_eyetracking.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+import pytest
+
+import mne
+from mne._fiff.constants import FIFF
+
+
+def test_set_channel_types_eyetrack(eyetrack_raw):
+    """Test that set_channel_types_eyetrack worked on the fixture."""
+    assert eyetrack_raw.info["chs"][0]["kind"] == FIFF.FIFFV_EYETRACK_CH
+    assert eyetrack_raw.info["chs"][1]["coil_type"] == FIFF.FIFFV_COIL_EYETRACK_POS
+    assert eyetrack_raw.info["chs"][0]["unit"] == FIFF.FIFF_UNIT_PX
+    assert eyetrack_raw.info["chs"][2]["unit"] == FIFF.FIFF_UNIT_NONE
+
+
+def test_convert_units(eyetrack_raw, eyetrack_cal):
+    """Test unit conversion."""
+    raw, cal = eyetrack_raw, eyetrack_cal  # shorter names
+
+    # roundtrip conversion should be identical to original data
+    data_orig = raw.get_data(picks=[0])  # take the first x-coord channel
+    mne.preprocessing.eyetracking.convert_units(raw, cal, "radians")
+    assert raw.info["chs"][0]["unit"] == FIFF.FIFF_UNIT_RAD
+    # Gaze was to center of screen, so x-coord and y-coord should now be 0 radians
+    np.testing.assert_allclose(raw.get_data(picks=[0, 1]), 0)
+
+    # Should raise an error if we try to convert to radians again
+    with pytest.raises(ValueError, match="Data must be in"):
+        mne.preprocessing.eyetracking.convert_units(raw, cal, "radians")
+
+    # Convert back to pixels
+    mne.preprocessing.eyetracking.convert_units(raw, cal, "pixels")
+    assert raw.info["chs"][1]["unit"] == FIFF.FIFF_UNIT_PX
+    data_new = raw.get_data(picks=[0])
+    np.testing.assert_allclose(data_orig, data_new)
```

Comment:
To save some chars we typically `from numpy.testing import assert_allclose` at the top